### PR TITLE
Tools folder with CBS automation solutions

### PR DESCRIPTION
**Pull request by romsok24**
After a successful deployment of your CBS environment instance, administrator will most probably want to automate some of the recurrent administration routines. 
Two interesting solutions that can help with it can be now explored in the [tools directory](https://github.com/epiphany-platform/central-build-system/tree/feat/tools/tools) of this repo.

### CODE REVIEW

File: tools/tekton-cleaner/tekton-cleanup.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: OnFailure
+          serviceAccount: tkn-cleaner
+          containers:
+            - name: kubectl
+              image: harbor.cbs.hasops.com/public/tkn:cleaner.05
```
Comment (mkyc) [inline tools/tekton-cleaner/tekton-cleanup.yaml:47]: is this correct?

**Review by mkyc — CHANGES_REQUESTED** [review]

File: tools/k8s-cert-manager/dns-01/dnschallenge_SP_create.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+JAKI_NS=cert-manager
```
[the remark this answers is not in the document]
Comment (romsok24) [reply tools/k8s-cert-manager/dns-01/dnschallenge_SP_create.sh]: The code was updated.  
Pls recheck.

File: tools/tekton-cleaner/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04 as posrednik
```
[the remark this answers is not in the document]
Comment (romsok24) [reply tools/tekton-cleaner/Dockerfile]: The code was updated.  
Pls recheck.

File: tools/tekton-cleaner/tekton-cleanup.yaml (hunk 1 above)
Comment (romsok24) [reply tools/tekton-cleaner/tekton-cleanup.yaml]: The code was updated.  
Pls recheck.

**Review by mkyc — APPROVED** [review]

**Pull request merged**